docs/api/structures/trace-categories-and-options.md

```diff
@@ -1,10 +1,6 @@
 # TraceCategoriesAndOptions Object
 
-* `categoryFilter` String – is a filter to control what category groups
-  should be traced. A filter can have an optional `-` prefix to exclude
-  category groups that contain a matching category. Having both included
-  and excluded category patterns in the same list is not supported. Examples:
-  `test_MyTest*`, `test_MyTest*,test_OtherStuff`, `-excluded_category1,-excluded_category2`.
+* `categoryFilter` String
```

Comment:
Not knowing anything about this, did its docs need to be completely rewritten?

Reply:
No, good catch I must have deleted this by accident at some point